src/kms/kmsAsymmetricSigningKey.ts

```diff
@@ -94,6 +95,7 @@ export class KMSAsymmetricSigningKey {
       };
       return verificationResult;
     } catch (error) {
+      console.error('Error while verifying from KMS:', error);
```

Comment:
[Nit] This doesn't seem grammatically apt. Can be rephrased to "Error from KMS while verifying: " Or "Error thrown during verification by KMS: "

Reply:
Similar pattern applicable at other error messages added.

Reply:
thanks for your suggestion, however, going with current message for now which also works and PR to merge urgently.